dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -380,7 +390,7 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent
                         .setPageMode(PageMode.LIVE)
                         .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ESP = "+html , html.contains("content3Spacontent2Spa"));
+        Assert.assertTrue("ESP = "+html , html.contains("content2Spa") && html.contains("content3Spa"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "ESP = " appears 5 times in this file; the first occurrence is on line 393](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5316867)